app/views/devise/mailer/unlock_instructions.html.erb

```diff
@@ -1,7 +1,12 @@
-<p>Hello <%= @resource.email %>!</p>
+<p>Dear <%= @resource.email %>!</p>
+
+<p>Thank you for using with CoinField.</p>
```

Comment:
Replace all "CoinField" with ENV.fetch('APP_NAME',  'Barong')

Reply:
you can't fetch things everywhere.

Reply:
We can have a helper method